crates/biome_js_analyze/src/lint/complexity/no_useless_fragments.rs

```diff
@@ -401,3 +415,7 @@ impl Rule for NoUselessFragments {
         }))
     }
 }
+
+fn contains_html_entity(s: &str) -> bool {
+    s.contains('&') && s.contains(';')
```

Comment:
This can cause many false positives, for example `<>something & and else;</>`. I would suggest to come up with a stronger solution to check HTML entities

Reply:
i refer the function from https://github.com/oxc-project/oxc/blob/main/crates/oxc_linter/src/rules/react/jsx_curly_brace_presence.rs#L520-L524,  the case `<>something & and else;</>` also need to preserve fragements

Reply:
It's still a stronger logic, because it checks their position too

Reply:
updated~